src/module/add.js

```diff
@@ -1,29 +1,69 @@
 const desc = document.querySelector('.inp');
-const el = document.createElement('div');
 const lists = document.querySelector('.lists');
 
 const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
 class Task {
-  constructor(desc, complete) {
+  constructor(desc) {
```

Comment:
- [ ] Great job so far, everything is going well. Please, you are missing the property completed and index value. as mentioned in the guidelines, By default, new tasks should have the property `completed` set to false and the property `index` set to the value of the new array length and Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. please kindly add and fix that. nice job.
- [optional] please ensure it is not possible to add an empty task(with no text) 


![Screenshot (946)_LI](https://user-images.githubusercontent.com/65298048/230515133-d2b8db38-6bfd-4afc-b904-967c5f551cbc.jpg)